query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverterTest.java

```diff
@@ -908,7 +958,24 @@ public void testQueryWithNulls() {
             + viewDefinition.getTenantIdColumn()
             + " = '"
             + TENANT_ID
-            + "' limit 15");
+            + "' limit 15",
+        viewDefinition);
+  }
+
+  @Test
+  public void testQueryWithAverageRateInOrderBy() {
```

Comment:
@avinashkolluru the integration test would be exactly the same as this. Should I keep this test also or remove this one?

Reply:
I think we should keep this as well. It is checking the expected PQL query.

Reply:
resolved